.gitignore

```diff
@@ -25,3 +25,5 @@ coverage
 *.njsproj
 *.sln
 *.sw?
+
+TODO.md
```

Comment:
https://marketplace.visualstudio.com/items?itemName=assisrMatheus.sidebar-markdown-notes might come in handy if you like to keep your todo list in code editor. And if you use vscode ;)

Reply:
Wow - nice one! Thanks!!